src/Marten/Events/Aggregation/TenantSliceGroup.cs

```diff
@@ -35,6 +34,12 @@ public interface ITenantSliceGroup<TId>
         /// <param name="events"></param>
         void AddEvents(TId id, IEnumerable<IEvent> events);
 
+        /// <summary>
+        /// Add events of type TEvent to streams by their self aggregate id (stream id/key)
+        /// </summary>
+        /// <param name="events"></param>
+        void AddEvents<TEvent>(IEnumerable<IEvent> events);
```

Comment:
Can you just rename the method to make it clear you're sorting events by their stream identity? I'd never guess what the method actually does by the method name here.